### ISSUE TITLE
Improvements in Asset Store docs

### ISSUE THREAD

**Issue by kazydek** [kyma-project/kyma#5897]
**Description**

After receiving constructive feedback on the existing Asset Store (AS), the following parts need a few tweaks:
1.  The [Overview](https://kyma-project.io/docs/components/asset-store/#overview-overview) doc should be improved - it must mention the main building blocks of AS (Asset & Bucket CRs, Asset & Bucket Controllers) and nicely sum up how AS works, having in mind that somebody can start and end with the Overview and should grasp AS's idea from the main doc.
2. Minio is mentioned in docs several times but the link to it is not added at the first mention but later in the doc - this is confusing. It should also be mentioned that Minio is an **external** , and not an in-house developed component/tool.
3. [Bucket lifecycle doc](https://kyma-project.io/docs/components/asset-store/#details-bucket-custom-resource-lifecycle-create-a-bucket-cr) - we mention the `{CR_NAME}-{GENERATED_ID}` location in the text and on the diagram but without explaining it, providing details of the naming convention used or giving any reference to the CRD docs that give more details - that should be explained/improved/referred to somehow.  Let's check if we describe the (cluster) bucket naming convention somewhere and, if not, let's add it and link to it.
4. Let's add a note somewhere about different costs of storing buckets depending on cloud providers and a recommendation that you should verify the payment policy with the given provider. Also, add a general recommendation that it is better to create fewer buckets per domain than a maltitude of buckets for each asset (the fewer buckets the better).
5. We link in Headless CMS (HCMS) docs to AS docs explaining the correlation between them, but we don't do it the other way round - add a link in AS docs to HCMS docs and explain how they are related.  
6. We lack a simple tutorial on how to run the Asset Store on your own and how to create a simple  CR.
7. Rename  Asset Store Controller to **Asset Store Controller Manager** in the [Metrics](https://kyma-project.io/docs/master/components/asset-store#metrics-asset-store-controller) doc, as this is the correct name.
8. Add info to the AS docs that we have Asset Store Controller Manager that manages the four controllers we use in AS (Asset Controller, ClusterAsset Controller, Bucket Controller, ClusterBucket Controller)

**Comment by kazydek** [comment]
This issue is related to and covered in https://github.com/kyma-project/rafter/issues/10.

**Comment by kazydek** [comment]
1. The overview was changed in general as we consolidated Asset Store and HCMS docs - I created separate docs for Rafter in Kyma and a general overview for Rfater which is based on details and technical explanation of Rafter in the `rafter` repository.
2. I mentioned twice in the overview docs (Rafter in Kyma and the general overview doc for Rafter) that Minio is an external solution + I added links to it).
3. `{CR_NAME}-{GENERATED_ID}` - I updated this ID (it changed a bit), provided an example in the text and on the diagram, and explained what it consists of (**Bucket custom resource lifecycle** doc).
4. Added the note into the consolidated tutorial on switching into Gateway mode.
5. No need for linking now since we have consolidated Rafter docs now.
6. I created a separate issue for that as it deserves a separate task -> https://github.com/kyma-project/kyma/issues/6292
7. That has been updated as a part of the consolidation task into Rafter Controller Manager.
8. That is explained in the Overview doc for Rafter.
